lib/evmone/instructions_calls.cpp

```diff
@@ -398,11 +404,24 @@ Result eofcreate(
     if (!check_memory(gas_left, state.memory, input_offset_u256, input_size_u256))
```

Comment:
This is weird that memory check happens _after_ we push 0 to the stack. Please check this.

Reply:
It's the same as what CREATE* is doing (stack is not observable after OOG)